src/resources/V2/Core/Events.ts

```diff
@@ -1,12 +1,58 @@
-// File generated from our OpenAPI spec
-
+// This file is manually maintained
 import {StripeResource} from '../../../StripeResource.js';
+
 const stripeMethod = StripeResource.method;
+
 export const Events = StripeResource.extend({
-  retrieve: stripeMethod({method: 'GET', fullPath: '/v2/core/events/{id}'}),
-  list: stripeMethod({
-    method: 'GET',
-    fullPath: '/v2/core/events',
-    methodType: 'list',
-  }),
+  constructor: function(...args: any) {
+    StripeResource.apply(this, args);
+  },
```

Comment:
I believe we needed to override the constructor coz we were setting the other resource on the Events resource. Since we dont do that anymore, we don't need to override the constructor right?

Reply:
oh that may be true.  The very important call here is that StripeResource.apply.  Let me make sure that gets called in the case where it isnt overridden.

Reply:
deleted the constructor, the generic constructor already calls apply the way we want